client/src/App.tsx

```diff
@@ -47,25 +45,11 @@ export default function App() {
     >
       <Box h="100%" w="100%" bg="black" tabIndex={0}>
         <Box pos="absolute" mt="xs" ml="xs" sx={{ zIndex: 10 }}>
-          <Button mr="xs" onClick={reset}>
-            Reset
-          </Button>
-          <Button
-            color="green"
-            onClick={start}
-            disabled={simulationStatus !== SimulationStatus.NotStarted}
-          >
-            Start
-          </Button>
+          <LaunchControls />
           <Box
             mt="xs"
             p="xs"
-            sx={(theme) => ({
-              backgroundColor:
-                theme.colorScheme === "dark"
-                  ? theme.colors.dark[5]
-                  : theme.colors.gray[0],
-            })}
+            sx={(theme) => stylesWithThemedBackgroundColor(theme)}
```

Comment:
This was maybe a little premature, but I centralized the logic for changing the bg color based on the theme. I also write the function (in styles.ts) in a way that allows you to add more CSS using the second argument to this function, with type safety (I think).
Ex: 
`sx={(theme) => stylesWithThemedBackgroundColor(theme, { color: pink })}`

I'm very open to a different API for this, it felt ergonomic enough for now.

Reply:
Yeah that looks great, I like the API